.github/workflows/linting.yml

```
@@ -0,0 +1,43 @@
+name: Linting
+
+on:
+  push:
+    branches: [ sj/ga-migrate-linting ]
+
+  
+jobs:
+  linting:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        
+      - run: sudo apt-get update
```

It's not a big deal, but for example [this answer](https://askubuntu.com/questions/445384/what-is-the-difference-between-apt-and-apt-get) suggests using `apt` rather than `apt-get`
```suggestion
      - run: sudo apt update
```